doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -261,6 +261,7 @@ Libraries for networking
 
   * Added A-GPS filtered ephemerides support, with ability to set matching threshold mask angle.
   * When filtered ephemerides is enabled, A-GPS assistance requests to cloud are limited to no more than once every two hours.
+  * Updated MQTT connection error handling to insure unacked ping and other errors result in a transition to the disconnected state so that reconnection can take place.
```

Comment:
This is quite a mouthful. Could we split this to two? For example: "Updated MQTT connection handling. Now, unacknowledged pings and other errors result in disconnected state so that..."